bots/crusader.js

```diff
@@ -121,7 +124,8 @@ crusader.takeTurn = (self) => {
     // }
 
     //Attack when number is more than 5
-    if(nearbyCrusaders.length >= 5 || a === 1){
+    var size = nearbyCrusaders.length + nearbyProphets.length
+    if(size >= 10 || a === 1){
```

Comment:
Why did you chose size>=10

Reply:
I just choose a random number, it can be more than 10